schema_utils.py

```diff
@@ -169,6 +176,18 @@ class Normalizers(object):
 
     @classmethod
     def get(cls, normalizer_id):
+        """Gets the value of the normalizer_id from the specified object.
```

Comment:
This is still incorrect -- the normalizer_id is the thing that's specified. (It's the argument that's passed in.)

Try: "Gets the normalizer method corresponding to the specified normalizer_id."